lib/src/main/java/tech/relaycorp/relaydroid/storage/KeyPairSerializer.kt

```diff
@@ -0,0 +1,27 @@
+package tech.relaycorp.relaydroid.storage
+
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+import java.security.KeyFactory
+import java.security.KeyPair
+import java.security.interfaces.RSAPrivateCrtKey
+import java.security.spec.PKCS8EncodedKeySpec
+import java.security.spec.RSAPublicKeySpec
+
+// TODO: Test
```

Comment:
Don't worry about testing this, I'll just implement it in the core lib: https://github.com/relaycorp/relaynet-jvm/issues/66

The core lib currently only supports deserialising public keys, so adding support for the whole key pair will be easy.